src/handbook/product/plan.md

```diff
@@ -53,14 +53,14 @@ The progress bars presented adjacent to each category are rough estimates, offer
 
 
 ### Data Visualization & Analytics
-![](https://geps.dev/progress/30)
+![](https://geps.dev/progress/40)
```

Comment:
Let's link to the dashboard 2.0 page, and get a better defintion for this hitting say 80%. I'd say that tagging a 1.0 would be getting us over 50% there?

Reply:
Yes, by the end of the year I expect to be at 60% 1.0 ready developed. Early next year, the 1.0 official release announcement.